Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -1350,6 +1350,31 @@ public SimpleMovingAverage SMA(Symbol symbol, int period, Resolution? resolution
             return simpleMovingAverage;
         }
 
+        /// <summary>
+        /// Creates a new Schaff Trend Cycle indicator
+        /// </summary>
+        /// <param name="symbol">The symbol for the indicator to track</param>
+        /// <param name="fastPeriod">The fast moving average period</param>
+        /// <param name="slowPeriod">The slow moving average period</param>
+        /// <param name="cyclePeriod">The signal period</param>
+        /// <param name="movingAverageType">The type of moving average to use</param>
+        /// <param name="resolution">The resolution</param>
+        /// <param name="selector">Selects a value from the BaseData to send into the indicator, if null defaults to the Value property of BaseData (x => x.Value)</param>
+        /// <returns>The SchaffTrendCycle indicator for the requested symbol over the specified period</returns>
+        public SchaffTrendCycle STC(Symbol symbol, int cyclePeriod, int fastPeriod, int slowPeriod, MovingAverageType movingAverageType = MovingAverageType.Exponential, Resolution? resolution = null, Func<IBaseData, decimal> selector = null)
+        {
+            var name = CreateIndicatorName(symbol, $"STC({cyclePeriod},{fastPeriod},{slowPeriod})", resolution);
```

Comment:
nit - `Name` isn't really being used here

Reply:
Whoops! Using name now in creation of STC